### PR TITLE
Blog post introducing kwok

### PR DESCRIPTION
**Pull request by wzshiming**
This PR adds a blog post for introducing kwok

mirror kubernetes/contributor-site#377

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Overall, I recommend this article for https://k8s.dev/blog/

However, we can mirror from that site to the main blog (with simultaneous publication).
To open a PR for the contributor website, visit https://github.com/kubernetes/contributor-site

**Comment by wzshiming** [comment]
Done, and wait for a review from the co-authors.

File: content/en/blog/_posts/2023-03-01-introducing-kwok/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Getting started
+
+If you are interested in trying out KWOK, please check its [documents] for more details.
+
+{{< figure src="/blog/2023/03/01/introducing-kwok/manage-clusters.svg" alt="Animation of a terminal showing kwokctl in use" caption="Using kwokctl to manage simulated clusters" >}}
+
+## Getting Involved
+
+If you're interested in future development and discussions on KWOK, you can get involved in SIG Scheduling through the following ways:
```
Comment (Huang-Wei) [inline content/en/blog/_posts/2023-03-01-introducing-kwok/index.md:65]: Again, don't need to mention scheduling, kwok is aimed for general use scenarios, not just scheduling

```suggestion
If you're interested in participating in future discussions or development related to KWOK, there are several ways to get involved:
```

File: content/en/blog/_posts/2023-03-01-introducing-kwok/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+- Slack: [#kwok], [#kwok-dev], [#sig-scheduling] (visit https://slack.k8s.io/ for a workspace invitation)
```
Comment (Huang-Wei) [inline content/en/blog/_posts/2023-03-01-introducing-kwok/index.md:67]:
```suggestion
- Slack: [#kwok] for general usage discussion, [#kwok-dev] for development discussion.
```

File: content/en/blog/_posts/2023-03-01-introducing-kwok/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- [SIG Scheduling]
+- Open Issues/PRs/Discussions in [sigs.k8s.io/kwok]
+
+We welcome feedback and contributions from anyone who wants to join us in this exciting project.
+
+[documents]: https://kwok.sigs.k8s.io/
+[sigs.k8s.io/kwok]: https://sigs.k8s.io/kwok/
+[#kwok]: https://kubernetes.slack.com/messages/kwok/
+[#kwok-dev]: https://kubernetes.slack.com/messages/kwok-dev/
+[SIG Scheduling]: https://github.com/kubernetes/community/blob/master/sig-scheduling/README.md
```
Comment (Huang-Wei) [inline content/en/blog/_posts/2023-03-01-introducing-kwok/index.md:77]: remove it.

**Comment by Huang-Wei** [comment]
> Overall, I recommend this article for [k8s.dev/blog](https://k8s.dev/blog/)

Ah, I _heard_ of this sort of dev blog but haven't got a chance to explore. Thanks @sftim . Very cool, this is definitely good to share some in-depth knowledge that is close to developers.

However, for this blog in particular, given it's more for introduction/promotion, IMO it fits the k8s.io blog more? WDYT?

> However, we can mirror from that site to the main blog (with simultaneous publication).

Mirroring in both sites doesn't sound _that_ necessary for me. But I do have some ideas like "using kwok to boost your Kubernetes contribution experience on your Macbook", which is a perfect match for the contributor blog site.

File: content/en/blog/_posts/2023-03-01-introducing-kwok/index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+## Why use KWOK?
+
+KWOK has several advantages:
+
+- **Speed**: You can create and delete clusters and nodes almost instantly, without waiting for boot or provisioning.
+- **Compatibility**: KWOK works with any tools or clients that are compliant with Kubernetes APIs, such as kubectl, helm, kui, etc.
+- **Portability**: KWOK has no specific hardware or software requirements. You can run it using pre-built images, once Docker or Nerdctl is installed. Alternatively, binaries are also available for all platforms and can be easily installed.
+- **Flexibility**: You can configure different node types, labels, taints, capacities, conditions, etc., and you can configure different pod behaviors, status, etc. to test different scenarios and edge cases.
+- **Performance**: You can simulate thousands of nodes on your laptop without great consumption of CPU or memory resources.
```
Comment (Huang-Wei) [inline content/en/blog/_posts/2023-03-01-introducing-kwok/index.md:34]: great -> significant

**Comment by sftim** [comment]
/hold

OK to unhold once https://github.com/kubernetes/contributor-site/pull/377 is good to go.

When would this be ready for review?

File: content/en/blog/_posts/2023-03-01-introducing-kwok/index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "Introducing KWOK: Kubernetes WithOut Kubelet"
+date: 2023-03-01
+slug: introducing-kwok
+---
+
+**Author:** Shiming Zhang (DaoCloud), Wei Huang (Apple), Yibo Zhuang (Apple)
```
Comment (sftim) [inline content/en/blog/_posts/2023-03-01-introducing-kwok/index.md:8]:
```suggestion
**Authors:** Shiming Zhang (DaoCloud), Wei Huang (Apple), Yibo Zhuang (Apple)
```

File: content/en/blog/_posts/2023-03-01-introducing-kwok/index.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "Introducing KWOK: Kubernetes WithOut Kubelet"
+date: 2023-03-01
+slug: introducing-kwok
+---
```
Comment (sftim) [inline content/en/blog/_posts/2023-03-01-introducing-kwok/index.md:6]:
```suggestion
canonicalUrl: https://kubernetes.dev/blog/2023/03/01/introducing-kwok/
---
```

**Review by sftim — COMMENTED** [review]
A couple of things to fix.

**Review by sftim — COMMENTED** [review]
/hold
/lgtm
/approve
